src/Framework/BuildMessageEventArgs.cs

```diff
@@ -227,6 +227,10 @@ public BuildMessageEventArgs
             this.endColumnNumber = endColumnNumber;
         }
 
+        public BuildMessageEventArgs(string message)
```

Comment:
Why did this need to be added?

Reply:
Oh you're right. Probably messed up some VS refactoring stuff